src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -1,6 +1,9 @@
 /** @jsx React.DOM */
 
+"use strict";
+
 var React = require("react/addons");
+var AppHealthComponent = require("../components/AppHealthComponent");
 
 var AppComponent = React.createClass({
   onClick: function () {
```

Comment:
Also a displayName here maybe? I know it is not your code, but it would be nice to get it on there :)